tests/virtctl/usbredir.go

```diff
@@ -68,112 +68,93 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 	const (
 		enoughMemForSafeBiosEmulation = "32Mi"
 		vmiRunTimeout                 = 90
-		delayToCleanup                = 100 * time.Millisecond
-		numTries                      = 3
 	)
 
 	Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component] A VirtualMachineInstance with usbredir support", func() {
```

Comment:
Can you apply the `SIG()` function since now it is a virtclt test?
```golang
Describe(SIG("[crit:medium][vendor:cnv-qe@redhat.com][lev...
```

Reply:
Also, looks like there are 2 nested `Describe` that can be merged

Reply:
> Also, looks like there are 2 nested Describe that can be merged

True, I think it is a left over of rewriting/removing things. Let me fix that.